docker/prod/docker-compose.yml

```diff
@@ -22,8 +22,8 @@ services:
     volumes:
       - static_dir:/andela_social/static
       - staticfiles:/andela_social/staticfiles
-      
-  client:
+
+  client: # not making a change here because I believe we will switch over when it's time
```

Comment:
Except maybe we want to have the `v2` deployed to a `andela-socials-staging.andela.com/v2` route sha.